addons/note/views/note_views.xml

```diff
@@ -167,7 +167,7 @@
         <field name="name">note.note.form</field>
         <field name="model">note.note</field>
         <field name="arch" type="xml">
-            <form string="Note" class="oe_form_nomargin o_note_form_view">
+            <form string="Note" class="o_note_form_view o_xxl_form_view">
```

Comment:
I don't think this class is meant to be hard-coded like that. I suggest to remove it.